jupyterhub/jupyterhub-idle-culler/base/buildconfig.yaml

```diff
@@ -0,0 +1,17 @@
+kind: BuildConfig
```

Comment:
We should create a repository for this with proper requirements or Pipfile and pre-build the image instead of building it on the fly